m3/src/main/java/com/uber/m3/tally/m3/M3Reporter.java

```diff
@@ -205,7 +238,16 @@ public void flush() {
             return;
         }
 
-        processors.forEach(Processor::scheduleFlush);
+        for (Processor processor : processors) {
+            processor.scheduleFlush();
+        }
+    }
+
+    // NOTE: This should only be used in tests
+    void flushNow() throws TException {
```

Comment:
Can we mark it with `@VisibleForTesting` ?

Reply:
We don't have Guava in the library